### DISCUSSION TITLE
Members of Communites and CFTs

### DISCUSSION THREAD

**Discussion by johannes-esr** [eclipse-score/score#90] [Technical Lead Circle]
Information to be added: full name & GitHub name
transfer of information to communities

# Operational (Tooling/Infrastructure) Community
- Alexander Lanin @AlexanderLanin
- Philip Partsch @PhilipPartsch
- Nikola Radakovic @nradakovic
- Dan Calavrezo @dcalavrezo-qorix
- Nicolae Dicu @nicu1989
- Daniel Woste @danwos 
- Sameer Srivastava @sameer6989
- Oliver Müller @mueller-o
- Jochen Hoenle, @hoe-jo 
- Maximillian Sören Pollak, @MaximilianSoerenPollak 
- Aymen Soussi
- Michal Mueller

# Software Development Process Community
- Philipp Ahmann @pahmann
- Philip Partsch @PhilipPartsch
- Alexander Schemmel @aschemmel-tech 
- Jochen Hönle, @hoe-jo 
- Volker Häussler @PandaeDo
- Satheesh R @SatheeshR79
- Markus Schu @masc2023
- Frank Scholter Peres @FScholPer  
- Daniel Kroehnert

# Testing Community
- Philipp Ahmann @pahmann
- Lukasz Tekieli, @ltekieli 
- Daniel Klotz @dneail1601
- Laszlo Bukai @LBukai
- Alberto Tejada @atm-acn
- Dragan Bjedov, @draganbjedov 
- Pascal Hirmer

# Software Architecture Community
- Arvid Sievert @arsibo
- Zoran Cutura @cuz9fe
- Ulrich Freund
- Oliver Emrich @olivembo
- Markus Bechter, @ramceb 
- Ulrich Huber, @LittleHuba 
- Lars Bauhofer @qor-lb 
- Vinod Reddy Gopi Reddy @vinodreddy-g 
- Nico Hartmann @HartmannNico 
- Martin Thiede @mthiede-acn2
- Martin Mössmer @mmoessmer-acn
- Armin Bartsch @armin-acn
- Fabian Frie @ffrie
- Alexander Warg
- Lutz Niebergall @LuNieb
- Guy Sagnes tba.

## FEO CFT
- Arvid Sievert @arsibo
- Ulrich Freund
- Alexander Lanin @AlexanderLanin 
- Mathias Danzeisen @MathiasDanzeisen 
- Markus Bechter, @ramceb 
- Ulrich Huber, @LittleHuba 
- Hanky Sjafrie @hanky-sensified
- Prabakaran Chinnasamy @prabakaranklst
- Pawel Rutka @pawelrutka1
- Lars Pöschel @poeschel
- Saumya Rai @Saumya-R
- Vinod Reddy Gopi Reddy @vinodreddy-g 
- Martin Thiede @mthiede-acn2
- Armin Bartsch @armin-acn
- Martin Mössmer @mmoessmer-acn
- Elena Ganter @theswiftfox
- Alexander Mohr @alexmohr
- Mark Schmitt @MarkSchmitt
- Frank Scholter Peres @FScholPer 

## IPC CFT
- Arvid Sievert @arsibo
- Oliver Emrich @olivembo
- Sebastian Schildt @SebastianSchildt 
- Danial Norouzi @nordanz
- Ulrich Huber, @LittleHuba 
- Martin Thiede @mthiede-acn2
- Armin Bartsch @armin-acn
- Martin Mössmer @mmoessmer-acn
- Elena Ganter @theswiftfox
- Alexander Mohr @alexmohr
- Mark Schmitt @MarkSchmitt
- Frank Scholter Peres @FScholPer 
- Vinod Reddy Gopi Reddy @vinodreddy-g 
- Dirk Linhos tba.

## Logging CFT
- Arvid Sievert @arsibo
- Philipp Ahmann @pahmann
- Nils Hoppe t.b.d.
- Gangadhara Maddikery Raghavendra t.b.d.
- Lars Pöschel @poeschel 
- Saumya Rai @Saumya-R
- Simon Gasse @simon-acc
- Paula Grosu tba.

## Persistency CFT
- Arvid Sievert @arsibo
- Sven Bachmann @sven
- Henri Bragge @hbragge-acn
- Johannes Glamsch @johannes-esr 
- Guy Sagnes tba.

**Comment by olivembo** [comment]
>     * Oliver Emrich
--> @olivembo

**Comment by LuNieb** [comment]
Paula Grosu  -->@palexuc
Guy Sagnes  --> @guysagnes
